Backend/UnitTesting/ErrorHandlingIT.cs

```diff
@@ -0,0 +1,291 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using WebApi_PointMap.Controllers;
+using DataAccessLayer.Models;
+using System.Net.Http;
+using System.Web.Http;
+using WebApi_PointMap.Models;
+using ServiceLayer.Services;
+using DataAccessLayer.Database;
+using System.Web.Http.Results;
+using System.Net;
+using System.Threading;
+using System.Diagnostics;
+using DTO.DTO;
+using System.Collections.Generic;
+
+namespace UnitTesting
+{
+    [TestClass]
+    public class ErrorHandlingIT
+    {
+        const string API_ROUTE_LOCAL = "http://localhost:58896";
+        PointController _pointController;
+        UserManagementController _umController;
+        TestingUtils _tu;
+        User newUser;
+        SessionService _ss;
+        DatabaseContext _db;
+
+        public ErrorHandlingIT()
+        {
+            _pointController = new PointController();
+            _umController = new UserManagementController();
+            _tu = new TestingUtils();
+            _ss = new SessionService();
+        }
+
+        [TestMethod]
+        public void GetUsersUnderManager_InvalidManagerId_412()

```

Comment:
Don't Test Manager services, need to refactor db to remove manager and claims from user